### ISSUE TITLE
New release with postgresql 10 support?

### ISSUE THREAD

**Issue by taybin** [ANXS/postgresql#345]
Postgres 10 support was merged into master almost a month ago.  Is a new release blocked on any other issues?  Or should the master branch be fine to use?

**Comment by gclough** [comment]
AFAIK the master is a good branch to use, but yes, it would make sense to do a new release with v10 support.  I'd ask @UnderGreen or @jlozadad to see when they consider a new version is appropriate.

**Comment by jlozadad** [comment]
I'll leave @UnderGreen drive that part since I have not done the releases yet.

**Comment by UnderGreen** [comment]
Released and imported to Ansible Galaxy.
